### ISSUE TITLE
Missing Geo Assets

### ISSUE THREAD

**Issue by Sostay** [hiddify/hiddify-next#240]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

![image](https://github.com/hiddify/hiddify-next/assets/6897274/cf73b930-ef2b-4af9-8b43-c8f4b178b1af)

### Reproduce the Bug

![image](https://github.com/hiddify/hiddify-next/assets/6897274/60fe992f-de92-4128-8e91-c7d5f04e6551)

### Expected Behavior

I can't use it anymore.……

### Version

V0.11.1

### Platform/OS

Windows

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by problematicconsumer** [comment]
Both routing asset's page and the error message are clear. We can't package all of these assets in the app, so it's up to the user to download them if need be. The error message clearly states that you should download (in menu) the missing asset or choose an existing one (by clicking on them!).

**Comment by Sostay** [comment]
> Both routing asset's page and the error message are clear. We can't package all of these assets in the app, so it's up to the user to download them if need be. The error message clearly states that you should download (in menu) the missing asset or choose an existing one (by clicking on them!).

The problem is that I can't download them, and I can't delete them ... Now I can't start the vpn service. Please give me a way to use it as before.

**Comment by Sostay** [comment]
why close this issue? i can't use the app now !???

**Comment by Sostay** [comment]
waiting someone can help me solve the problem……

**Comment by problematicconsumer** [comment]
@Sostay Maybe my explanation wasn't clear enough. Your active routing assets (geoip and geosite) must exist! You have marked assets as active that don't exist. If you want to use default assets (which are bundled with the app, hence no need to download) simply select the 2 items in the top of the list, they will be marked as active.

**Comment by Sostay** [comment]
> @Sostay Maybe my explanation wasn't clear enough. Your active routing assets (geoip and geosite) must exist! You have marked assets as active that don't exist. If you want to use default assets (which are bundled with the app, hence no need to download) simply select the 2 items in the top of the list, they will be marked as active.

I basically understand, but I know you didn't provide a solution, but now I have completely reset the app, starting from scratch ...

**Comment by problematicconsumer** [comment]
> you should download (in menu) the missing asset or choose an existing one (by clicking on them!).

This is from my initial response.
I understand that the UI might not be intuitive enough. I will add a new card on the top of Routing Assets page to warn user when selecting assets with files missing.
